src/main/java/org/eln2/mc/extensions/DataBuilderExtensions.kt

```diff
@@ -0,0 +1,111 @@
+package org.eln2.mc.extensions
+
+import org.eln2.libelectric.sim.electrical.mna.component.Capacitor
+import org.eln2.libelectric.sim.electrical.mna.component.Inductor
+import org.eln2.libelectric.sim.electrical.mna.component.Resistor
+import org.eln2.libelectric.sim.electrical.mna.component.VoltageSource
+import org.eln2.mc.utility.*
+import kotlin.math.abs
+
+object DataBuilderExtensions {
+    private class InternationalSystemEntry(override val label : String, val number : Double, val decimals : Int, val unit : String)
+        : DataBuilder.Entry(label, SuffixConverter.convert(abs(number), unit, decimals))
```

Comment:
Decimals are nice but Cameron and I were thinking we should use significant digits instead like old Electrical Age did.